ocs_ci/deployment/deployment.py

```diff
@@ -652,3 +667,131 @@ def create_catalog_source(image=None, ignore_upgrade=False):
     )
     # Wait for catalog source is ready
     catalog_source.wait_for_state("READY")
+
+
+def setup_local_storage():
+    """
+    Setup the necessary resources for enabling local storage.
+
+    """
+    # Get the worker nodes
+    workers = get_typed_nodes(node_type='worker')
+    worker_names = [worker.name for worker in workers]
+    worker_names_str = " ".join(worker_names)
+    logger.debug("Workers: %s", workers)
+
+    # Label the worker nodes
+    logger.info("Labeling worker nodes")
+    run_cmd(
+        f"oc label nodes {worker_names_str} "
+        f"{constants.OPERATOR_NODE_LABEL}"
+    )
+
+    logger.info("Retrieving local-storage-operator data from yaml")
+    lso_data = list(templating.load_yaml(
+        constants.LOCAL_STORAGE_OPERATOR, multi_document=True
+    ))
+    # Update local-storage-operator subscription data with channel
+    for data in lso_data:
+        if data['kind'] == 'Subscription':
+            data['spec']['channel'] = config.ENV_DATA.get('ocs_version')
```

Comment:
This is probably related to version of OCP and not OCS. Please ask in the KNIP and for example for un GAed OCP version like 4.4.

Reply:
I haven't received confirmation of this yet but I swapped to using the ocp version instead.